docs/architecture/raft.rst

```diff
@@ -54,6 +54,10 @@ When all the nodes in the cluster are upgraded to ScyllaDB Open Source 5.5 or Sc
 Verifying that the Raft upgrade procedure finished successfully
 ========================================================================
 
+.. note::
+    After the procedure described in this section finishes, if your cluster is running ScyllaDB Open Source 6.0 then you must perform manual action in order to enable consistent topology changes.
```

Comment:
> I think this is implicitly assumed by the fact that we version documentation

Right. However, I do see that we explicitly mention versions in some docs, e.g.:

```rst
When all the nodes in the cluster are upgraded to ScyllaDB Open Source 5.5 or ScyllaDB Enterprise 2024.2, the cluster will start the **Raft upgrade procedure**.
```

therefore I thought that this would be OK, but in the end I'm not sure. @annastuchlik , what's the best practice here?